style.css

```diff
@@ -191,7 +193,8 @@ header {
 	align-items: center;
 }
 
```

Comment:
Don't forget to apply the fonts to the form inputs and buttons. Right now these are being set to sans-serif by normalize.css.

Reply:
applied --body-font to email form